shotgun_api3/shotgun.py

```diff
@@ -1473,10 +1497,13 @@ def update_project_last_accessed(self, project, user=None):
         record = self._call_rpc("update_project_last_accessed_by_current_user", params)
         result = self._parse_records(record)[0]
 
-
-    def _get_session_token(self):
-        """Hack to authenticate in order to download protected content
-        like Attachments
+    def get_session_token(self):
```

Comment:
Yes, that's perfectly fine. I simply added my comment to state that it was still important to keep it in, since previous comments on that method indicated that you were not sure it would still be needed.